pages/markets/[marketid].tsx

```diff
@@ -156,35 +156,22 @@ const Market: NextPage<{
         <div>
           <MarketImage
             image={indexedMarket.img}
-            alt={`Image depicting ${question}`}
+            alt={`Image depicting ${indexedMarket.question}`}
             size="120px"
             status={indexedMarket.status}
             className="mx-auto"
           />
           <MarketHeader
-            question={question}
-            status={indexedMarket.status}
-            tags={indexedMarket.tags}
-            starts={starts}
-            ends={ends}
+            market={indexedMarket}
+            resolvedOutcome={marketSdkv2?.resolvedOutcome}
+            report={marketSdkv2?.report as Report}
```

Comment:
Any way to avoid the casting here? Maybe there is a type we can pull from the sdk

Reply:
I found a type called `MarketReport` that would work but it's in here `@zeitgeistpm/indexer/src/graphql/sdk` and that was causing a build error so I tweaked a few things inside `useEffect` to make it work